kvmagent/kvmagent/test/test_fence_vm_on_suspect_host.py

```
@@ -0,0 +1,299 @@
+            rsp.qemuConfirmedDead = True
+        elif rc == 2:
+            rsp.qemuStillAlive = True
+            rsp.success = False
+            rsp.error = "qemu still alive on %s after force-destroy attempt" % target_ip
+        else:
+            rsp.targetHostUnreachable = True
+    finally:
+        try:
+            _os.remove(sshpass_file)
+        except Exception:
+            pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)